python/datashaper/datashaper/workflow/workflow.py

```diff
@@ -45,6 +46,13 @@
 DEFAULT_INPUT_NAME = "source"
 
 
+class PandasDtypeBackend(str, Enum):

```

Comment:
What's the default?

Reply:
`pandas_dtype_backend: PandasDtypeBackend = PandasDtypeBackend.NUMPY_NULLABLE,` in the constructor of the Workflow

Reply:
Is this explicit in any docstring?